generators/server/templates/src/test/java/package/service/UserServiceIntTest.java.ejs

```diff
@@ -101,7 +100,7 @@ import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 <%_ } _%>
-<%_ if (authenticationType !== 'oauth2' && (databaseType === 'sql' || databaseType === 'mongodb' || databaseType === 'couchbase')) { _%>
```

Comment:
why is sql removed from here?

Reply:
Hi Deepu,
Thanks for the review.
I removed this condition because it deal with Optional import that is already set below : 
`<%_ if (databaseType === 'sql') { _%>
import java.time.LocalDateTime;
import java.util.Optional;
<%_ } _%>`

For SQL in all authenticationType cases we need to import "Optional". It remove a redundant import.

Reply:
ok